Source/Scene/GltfVertexBufferLoader.js

```diff
@@ -37,6 +37,7 @@ import ComponentDatatype from "../Core/ComponentDatatype.js";
  * @param {Boolean} [options.asynchronous=true] Determines if WebGL resource creation will be spread out over several frames or block until all WebGL resources are created.
  * @param {Boolean} [options.dequantize=false] Determines whether or not the vertex buffer will be dequantized on the CPU.
  * @param {Boolean} [options.loadAsTypedArray=false] Load vertex buffer as a typed array instead of a GPU vertex buffer.
+ * @param {Boolean} [options.loadFor2D=false] Load vertex buffer as both a buffer and typed array, the latter of which will be projected to 2D.
```

Comment:
Similarly to the indices, does it make sense to rephrase these lower-level options as `loadTypedArray` and `loadBuffer`? If so, such options would not be mutually exclusive.